deployment-guide/upgrade-guides/upgrading-cinchy-versions/kubernetes-upgrades/v5.7-kubernetes.md

```diff
@@ -0,0 +1,203 @@
+
+```bash
+bash create.sh
+```
+
+#### CIDR range lower than 10.10.0.0/22
+
+{% hint style="danger" %}
+This section is only applicable to Azure deployments.
+{% endhint %}
+
+If you have **`10.10.0.0/22 CIDR` range or smaller**, then you won't be able to upgrade AKS version to 1.25.x. The `10.10.0.0/22` CIDR range gives you 1024 IP addresses, which aren't enough to run more than 4 worker nodes. Most customers already run 3 worker nodes and the upgrade process starts another 3 nodes, which will cause a failure.
+
+The values below gives the suggested IP address CIDR range. Cinchy recommends to make your own choice based on your needs. Update these values in your `deployment.json` file:
+
+```json
+          "address_space": "10.10.0.0/19",
+          "subnet_prefix": "10.10.0.0/20"
+          "net_profile_service_cidr": "10.1.4.0/22",
+          "net_profile_dns_service_ip": "10.1.4.2",
+```
+
+### Delete ArgoCD apps
+
+Make sure you are connected to the appropriate cluster. Before you start the upgrade process of AKS, you must delete all your apps from ArgoCD which will delete Cinchy apps and custom components from ArgoCD, which includes `load balancer` as well `kafka-cluster (Warning)`.
+
+To delete Cinchy apps, cluster components and ArgoCD:
+
+1. From terminal change directory to **cinchy.argocd** and run the commands below sequentially. Make sure to change your cluster(directory) name and environment name in below commands:
+
+    ```bash
+    kubectl delete -k environment_kustomizations/CLUSTER_NAME/cluster_components
+    kubectl delete -k environment_kustomizations/hub-nonprod/ENVIRONMENT_NAME/cinchy
+    kubectl delete -k argocd
+    ```
+1. Verify the deletion of pods from that specific namespace with the `kubectl get pods -A` command. 
+1. If cluster components deletion takes longer than 10 minutes, run the command below. Check all the pods except `Kubernetes default namespace(kube-system)` pods.
+    ```bash
+    kubectl patch app $(kubectl get applications -n argocd --no-headers | awk '{print $1}') -p '{"metadata":{"finalizers":null}}' --type=merge -n argocd
+    ```
+
+### Deploy
+
+1. Change the AKS version in DevOps automation tools `deployment.json `against `kubernetes_version` and `orchestrator_version` key values. From a shell/terminal, go to the **cinchy.devops.automations** directory location and execute the following command:
+    ```bash
+    dotnet Cinchy.DevOps.Automations.dll "deployment.json"
+    ```
+1. In the **Terraform > Azure** directory, a new folder named `aks_cluster` should be updated with the AKS version. In that directory is a subdirectory with the same name as the newly updated cluster.
+1. Launch a shell/terminal with the working directory set to the cluster directory within the cinchy.terraform repo and run `az login`.
```

Comment:
This was changed as `az login` was on a different line with no reference to running it. However earlier in the doc `az login` was mentioned as a command, so I adjusted it. @jadhavnitind this should be approved by you.